src/modules/core.js

```diff
@@ -1642,7 +1642,7 @@
 		 * @param {number} revId Revision ID of the draft that is being accepted
 		 * @param {boolean} isBiography Value of the "Is the article a biography?" check box
 		 * @param {Array} newWikiProjects Value of the "Add WikiPrjects" part of the form. The <input> is a chips interface called jquery.chosen. Note that if there are existing WikiProject banners on the page, the form will auto-add those to the "Add WikiProjects" part of the form when it first loads.
-		 * @param {string} lifeStatus Value of "Is the subject alive?" dropdown list ("unknown", "living", "dead")
+		 * @param {string} lifeStatus Value of "Is the subject alive?" dropdown list ("unknown", "blp", "dead")
```

Comment:
Internally in AFCH (but not in the wikicode), can we keep using "living" as the value? I find living to be more readable.